diff --git a/pyproject.toml b/pyproject.toml
index cb0b0e7..5d94f7a 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -29,7 +29,7 @@ keywords = ["msilib"]
 license = "PSF-2.0"
 license-files = ["LICENSE"]
 readme = "README.md"
-requires-python = ">=3.13"
+requires-python = ">=3.10"
 
 [[project.authors]]
 name = "Marcelo Duarte"
diff --git a/tests/test_msilib.py b/tests/test_msilib.py
index 3dbdd40..b65f75a 100644
--- a/tests/test_msilib.py
+++ b/tests/test_msilib.py
@@ -1,6 +1,7 @@
 """Test suite for the code in msilib."""
 
 import os
+import sys
 
 import pytest
 
@@ -97,6 +98,9 @@ def test_directory_start_component_keyfile(db, tmp_path) -> None:
         msilib._directories.clear()  # noqa: SLF001
 
 
+@pytest.mark.skipif(
+    sys.version_info[:2] < (3, 13), reason="Only for Python 3.13+"
+)
 def test_large_package(db, tmp_path) -> None:
     root_dir = tmp_path / "root"
     root_dir.mkdir()
